disco_aws_automation/disco_storage.py

```diff
@@ -237,47 +241,59 @@ def create_ebs_snapshot(self, hostclass, size):
         :param hostclass:  The hostclass that uses this snapshot
         :param size:  The size of the snapshot in GB
         """
-        zones = self.connection.get_all_zones()
+        zones = throttled_call(self.connection.get_all_zones)
         if not zones:
             raise VolumeError("No availability zones found.  Can't create temporary volume.")
         else:
             zone = zones[0]
 
             def _destroy_volume(volume, raise_error_on_failure=False):
-                if self.connection.delete_volume(volume_id=volume.id):
+                if throttled_call(self.connection.delete_volume, volume_id=volume.id):
                     logging.info("Destroyed temporary volume %s", volume.id)
                 elif raise_error_on_failure:
                     raise VolumeError("Couldn't destroy temporary volume %s", volume.id)
                 else:
                     logging.error("Couldn't destroy temporary volume %s", volume.id)
 
             try:
-                volume = self.connection.create_volume(size=size, zone=zone)
+                volume = throttled_call(self.connection.create_volume, size=size, zone=zone)
                 logging.info("Created temporary volume %s in zone %s.", volume.id, zone.name)
                 wait_for_state(volume, 'available', state_attr='status')
                 snapshot = volume.create_snapshot()
                 snapshot.add_tag('hostclass', hostclass)
+                snapshot.add_tag('env', self.environment_name)
                 logging.info("Created snapshot %s from volume %s.", snapshot.id, volume.id)
             except Exception:
                 _destroy_volume(volume)
                 raise
             else:
                 _destroy_volume(volume, raise_error_on_failure=True)
 
-    def get_snapshots(self, hostclasses):
+    def get_snapshots(self, hostclasses=None):
         """
```

Comment:
The method supports passing `None` to get all of the snapshots. I thought we could make it the default instead of making people explicitly pass in `None`.